gaztec/src/components/contract/components/deployContractDialog.tsx

```diff
@@ -0,0 +1,184 @@
+import DialogTitle from "@mui/material/DialogTitle";
+import Dialog from "@mui/material/Dialog";
+import {
+  AccountWalletWithSecretKey,
+  ContractDeployer,
+  Fr,
+} from "@aztec/aztec.js";
+import {
+  Autocomplete,
+  Button,
+  CircularProgress,
+  FormControl,
+  FormGroup,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  css,
+} from "@mui/material";
+import { useContext, useEffect, useState } from "react";
+import { ContractInstanceWithAddress, PublicKeys } from "@aztec/circuits.js";
```

Comment:
Bit strange that we need this separate import for this

Reply:
It was not, merged them and got rid of `circuit-types`. `circuits.js` still needed for key derivation, but hopefully we can get rid of it soon ™️